documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -46,6 +44,15 @@ Limits can be overridden by xref:con-configuring-client-quotas-str[user-specific
 * CPU utilization limits for each client can be set as a percentage of the network threads and I/O threads on a per-broker basis.
 * The number of concurrent partition creation and deletion operations (mutations) allowed per second can be set on a per-broker basis.
 
+When using the default Kafka quotas plugin, quotas are applied to all users. 
+This includes internal users such as the Topic Operator and Cruise Control, which may impact their operations.
+To avoid unduly limiting internal users, consider tuning the quotas effectively.
+
+For example, configuring a controller mutation rate quota may result in the Topic Operator being throttled during topic creation or deletion operations.
```

Comment:
Is this a warning about how the Kafka quotas plugin automatically sets a quota for the TO user, affecting things like the controller mutation rate? If so, could we say something like: "For example, the quota automatically applied to the Topic Operator by the Kafka quotas plugin could choke the controller mutation rate resulting in" ... and "Therefore, it is important to understand the minimal quotas required by Topic Operator to function properly and set the quota explicitly to avoid such issues"